$NetBSD: patch-ai,v 1.2 2008/02/06 07:53:32 bjs Exp $

Beginnings of emacs support, not enabled yet.  We must use mk/emacs.mk.

--- contrib/emacs/Makefile.orig	2008-02-01 23:09:01.000000000 -0500
+++ contrib/emacs/Makefile
@@ -12,8 +12,8 @@ RM ?= rm -f
 all: $(ELC)
 
 install: all
-	$(INSTALL) -d $(DESTDIR)$(emacsdir)
-	$(INSTALL_ELC) $(ELC:.elc=.el) $(ELC) $(DESTDIR)$(emacsdir)
+	${BSD_INSTALL_DATA_DIR} $(DESTDIR)$(emacsdir)
+	${BSD_INSTALL_DATA} $(ELC:.elc=.el) $(ELC) $(DESTDIR)$(emacsdir)
 
 %.elc: %.el
 	$(EMACS) -batch -f batch-byte-compile $<
